src/views/PokeList.js

```diff
@@ -0,0 +1,75 @@
+import React, { useEffect, useState } from 'react';
+import * as pokemonServices from '../services/pokemons.service';
```

Comment:
Easier and more clean if you just explicit import what you need. Is like destructuring an object.